### PR TITLE
Reduce false positives from Sonatype Lift

### PR DESCRIPTION
**Pull request by jlstephens89**
Fixes 

- Ignore SkyWalking-ui/* : https://github.com/apache/skywalking/pull/8736#issuecomment-1079846668
- Disable NonAtomicVolatileUpdate : https://github.com/apache/skywalking/pull/8736#issuecomment-1079850139
- Ignore license files (and therefore incorrect JSC_PARSE_ERROR warning) : https://github.com/apache/skywalking/pull/8736#issuecomment-1079850428
- Disable EqualsGetClass https://github.com/apache/skywalking/pull/8736#issuecomment-1079852283

Example run: https://lift.sonatype.com/results/github.com/doddi/skywalking/01FZDGD4AA1TYCH777857MMYF5?tab=results

### CODE REVIEW

File: .lift/config.toml [hunk 1]
```diff
@@ -1,3 +1,8 @@
 jdk11 = true
-ignoreRules = [ "THREAD_SAFETY_VIOLATION", "MissingSummary", "SameNameButDifferent" ]
+ignoreRules = [ "THREAD_SAFETY_VIOLATION", "MissingSummary", "SameNameButDifferent", "NonAtomicVolatileUpdate", "EqualsGetClass"]
 setup = ".lift/setup.sh"
+ignoreFiles = """
+skywalking-ui/
+apm-protocol/apm-network/target/
```
Comment (wu-sheng) [inline .lift/config.toml:6]: Could we ignore folders lik `**/generated-sources/**`? All protoc generated files are there, but in different folder paths.

Comment (jlstephens89) [reply .lift/config.toml:6]: @wu-sheng Do you mean instead of the target directory or as well as?

Comment (jlstephens89) [reply .lift/config.toml]: Makes sense. 
I've added generated sources https://github.com/apache/skywalking/pull/8769/commits/9902e290de934f211e88ffbd2d4e99bb2f01d4f5

For safety I've also excluded any target directory 
https://github.com/apache/skywalking/pull/8769/commits/5b398d558f19e3419bf7de79bdfade3330c46f21

**Review by wu-sheng — APPROVED** [review]
Once lift check passed, we could merge this first.

**Pull request merged**